src/tqec/plaquette/plaquette.py

```diff
@@ -148,19 +164,21 @@ def __init__(
                 orientation of PlaquetteOrientation.UP will generate a plaquette
                 with its rounded side pointing up.
         """
-        super().__init__(
-            RoundedPlaquette.get_data_qubits(orientation)
-            + RoundedPlaquette.get_syndrome_qubits(),
-            circuit,
-        )
+        super().__init__(circuit)
+        self._orientation = orientation
 
-    _data_qubits: list[PlaquetteQubit] = [
-        PlaquetteQubit(Position(0, 0)),
-        PlaquetteQubit(Position(2, 0)),
-        PlaquetteQubit(Position(0, 2)),
-        PlaquetteQubit(Position(2, 2)),
-    ]
-    _syndrome_qubits = [PlaquetteQubit(Position(1, 1))]
+    @staticmethod
+    def _get_data_qubit_side_from_plaquette_orientation(
+        orientation: PlaquetteOrientation,
+    ) -> PlaquetteSide:
```

Comment:
This relationship can be defined within the `PlaquetteOrientation` or `PlaquetteSide` enum.